descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,45 @@
   "id": "@artifactId@-@version@",
   "name": "Folio mod-circulation-bff module",
   "provides": [
```

Comment:
We're adding endpoints that will be calling other endpoints in mod-circulation and mod-tlr. Do we already have dependencies on their corresponding interfaces or do we need to add them?

Reply:
added dependencies